livekit-agents/livekit/agents/pipeline/pipeline_agent.py

```diff
@@ -717,7 +717,9 @@ def _commit_user_question_if_needed() -> None:
                 )
 
             if tool_calls:
-                extra_tools_messages.append(ChatMessage.create_tool_calls(tool_calls))
+                extra_tools_messages.append(
+                    ChatMessage.create_tool_calls(tool_calls, content=collected_text)
```

Comment:
note small functional change here - this'll be "" instead of None if there are no messages in the context (can change it but wasn't sure it mattered)